src/commands/job/execute.js

```diff
@@ -156,3 +165,27 @@ export async function execute(
 
   return result
 }
+
+async function displayStatus(submittedJob, statusOfJob, result = {}) {
+  const adapterStatus =
+    submittedJob && submittedJob.state ? submittedJob.state : 'Not Available'
+  const status =
+    adapterStatus === 'Not Available'
+      ? `Job Status: ${adapterStatus}\n\nDetails: ${result}`
+      : `Job Status: ${adapterStatus}`
+
+  if (adapterStatus === 'completed') displayResult(null, null, status)
+  else displayResult(result, status, null)
+
+  if (typeof statusOfJob === 'string') {
```

Comment:
The `JSDoc` comment above indicates that `statusOfJob` is a boolean.
Are we 'reusing' that to pass in a path? I also feel like the variable could be named differently to better reflect its intent.

Reply:
JSDoc needs to be updated, since approach is changed
Yes it's not boolean flag anymore.